packages/lab/src/layout/LayerLayout/LayerLayout.tsx

```diff
@@ -0,0 +1,111 @@
+import { forwardRef, HTMLAttributes, useState, useEffect } from "react";
+import cx from "classnames";
+
+import { makePrefixer, Breakpoints } from "@jpmorganchase/uitk-core";
+import { Scrim, ScrimProps } from "@jpmorganchase/uitk-lab";
+import "./LayerLayout.css";
+import { useChangeView } from "../utils";
+
+export const LAYER_POSITION = [
+  "center",
+  "left",
+  "top",
+  "right",
+  "bottom",
+] as const;
+
+export type LayerPosition = typeof LAYER_POSITION[number];
+
+export interface LayerLayoutProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Display or hide the scrim.
+   */
+  displayScrim?: boolean;
+  /**
+   * Defines the layer position within the screen.
+   */
+  position?: LayerPosition;
+  /**
+   * Breakpoint at which the layer will become fullscreen.
+   */
+  fullScreenAtBreakpoint?: keyof Breakpoints;
+  /**
+   * Disable all animations.
+   */
+  disableAnimations?: boolean;
+  /**
+   * Display or hide the component.
+   */
+  open?: boolean;
+  /**
+   * Props to be passed to the Scrim component.
+   */
+  scrimProps?: ScrimProps;
+}
+
+const withBaseName = makePrefixer("uitkLayerLayout");
+
+export const LayerLayout = forwardRef<HTMLDivElement, LayerLayoutProps>(
+  function LayerLayout(props, ref) {
+    const {
+      children,
+      className,
+      displayScrim = true,
+      position = "center",
+      fullScreenAtBreakpoint = "sm",
+      disableAnimations = false,
```

Comment:
Not sure if it's a big deal but I noticed that if you toggle disableAnimations on and off while the Layer is 'open' then the Layer will show the enter animation each time disableAnimations is set to false.

Reply:
Should be fixed now